Sources/SKTestSupport/SkipUnless.swift

```diff
@@ -106,7 +106,8 @@ public enum SkipUnless {
       let response = try unwrap(
         await testClient.send(DocumentSemanticTokensRequest(textDocument: TextDocumentIdentifier(uri)))
       )
-      let tokens = [SyntaxHighlightingToken](lspEncodedTokens: response.data)
+
+      let tokens = SyntaxHighlightingTokens(lspEncodedTokens: response.data)
```

Comment:
Maybe I’m missing something but I don’t think this build with the array comparison below. Could you make sure that the package builds and passes tests on your machine?

Reply:
As you expected, it didn't passess the tests. which one do you prefer, implementing array comparison as a method inside ```SyntaxHighlightingTokens``` struct or just compare tokens property directly since its an array and has built-in method

Reply:
Let’s just compare the arrays similar to what we do in `SemanticTokensTests.swift`.